src/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+## crypto plugins
+
+set(crypto_plugin_dir ${CMAKE_INSTALL_PKGLIBDIR}/crypto)
+set(crypto_plugins_deps "")
+
+if(HAVE_INTEL AND HAVE_BETTER_YASM_ELF64 AND (NOT APPLE))
+  add_subdirectory(isa-l)
+  list(APPEND crypto_plugins_deps ceph_crypto_isal)
+endif(HAVE_INTEL AND HAVE_BETTER_YASM_ELF64 AND (NOT APPLE))
```

Comment:
could you remove remove the `HAVE_INTEL AND HAVE_BETTER_YASM_ELF64 AND (NOT APPLE)` from the `endif()` statement? as it's a very short `if` block.